README.md

```diff
@@ -41,96 +62,271 @@ catch (Exception $e) {
 }
 ```
 
-This will give you JSON object containing details of the Payment Request that was just created.
+This will give you JSON object containing details of the Payment Request that was just created with `longurl` key provide you instamojo-payment-url.
```

Comment:
Grammatically incorrect. Please rectify

Reply:
@mojodhanesh  you want method name to be paymentRequestCreate instead of createPaymentRequest ?

Reply:
Read the last 7 words, they make no grammatical sense